doc/howto/grafana.md

```diff
@@ -82,3 +82,27 @@ To set the `loki.instance` configuration key, run the following command:
 You can check that setting via:
 `lxc config get loki.instance`
 ```
+
+## Scripted setup and LXD UI integration
+
+As an alternative to the manual steps above, we provide a script to set up the Grafana dashboard. This only supports a single-node LXD installation.
+
+1. Launch a new instance on your LXD server:
+
+       lxc launch ubuntu:20.04 grafana --project default
+
+1. Run the following commands to download and execute the script to set up Grafana on the `grafana` instance:
+
+       curl -s https://raw.githubusercontent.com/canonical/lxd-ui/refs/heads/main/public/assets/scripts/setup-grafana.sh -o /tmp/setup-grafana.sh
```

Comment:
Can we move this into the lxd repo so we arent askimg users to execute a script outside of the main repo.